packages/govern-lib/README.md

```diff
@@ -0,0 +1,27 @@
+# GovernLib
+
+GovernLib is a library that allows Node.js apps and browsers to access the Govern related data in a unified way.
```

Comment:
This sentence could confuse devs cause they aren't sure anymore if they should use the lib or govern.js.. We probably could call this package also ``govern-shared`` 🤷‍♂️

Reply:
True, I will rephrase it. `govern-shared` doesn’t sound like something you could run from an app directly. I was trying to find a name similar to `govern-core`, that we can’t use since we already do for the contracts.

Reply:
I'm not sure if we really want to let our users directly use the ``core`` functionalities of the ``govern-server`` package. I think they should use them through the ``govern-server`` package with a thin abstraction layer. This would give us a lot of freedom in the future of how we want to have our ``core`` functionalities be architected.